35-Merge-overlapping.js

```diff
@@ -0,0 +1,16 @@
+/* Dia 35
+https://www.codewars.com/kata/61c78b57ee4be50035d28d42/train/javascript
+Este kata requiere que escribas una función que fusione dos cadenas. Para ello, fusiona el final de la primera cadena con el comienzo de la segunda cuando coinciden exactamente.
+*/
```

Comment:
En la descripción, sería util incluir ejemplos. En el texto del consigne incluye:
```
"abcde" + "cdefgh" => "abcdefgh"
"abaab" + "aabab" => "abaabab"
"abc" + "def" => "abcdef"
"abc" + "abc" => "abc"
```

También hay una nota importante: `The algorithm should always use the longest possible overlap.` => `El algoritmo debe siempre ocupar la superposición más larga.`

Alternativamente, puedes hacer pruebas que muestran el resultado esperado, dado las entradas. Esto sería ideal, porque mejorará tu proceso de desarrollo y practica. Te puedo ayudar configurar eso.

Reply:
Gracias lo voy a tener en cuenta para los próximos ejercicios.